rapid7_insightvm_cloud/icon_rapid7_insightvm_cloud/actions/stop_scan/action.py

```diff
@@ -17,20 +17,14 @@ def __init__(self):
     def run(self, params={}):
         scan_id = params.get(Input.ID)
         url = f"scan/{scan_id}/stop"
-        response = self.connection.ivm_cloud_api.call_api(url, "POST")
-        if response[0] == 202 and response[1] is None:
-            response = self.connection.ivm_cloud_api.call_api("scan/" + scan_id, "GET")
-            if response[1].get("status") == "Stopped":
-                return {Output.SUCCESS: True}
-            else:
-                return {
-                    Output.SUCCESS: False,
-                    Output.STATUS_CODE: 400,
-                    Output.MESSAGE: f"Failed to stop scan with ID '{scan_id}'. Status of scan is {response[1].get('status')}",
-                }
+        self.connection.ivm_cloud_api.call_api(url, "POST")
+
+        response = self.connection.ivm_cloud_api.call_api("scan/" + scan_id, "GET")
+        if response.get("status") == "Stopped":
+            return {Output.SUCCESS: True}
         else:
             return {
                 Output.SUCCESS: False,
-                Output.STATUS_CODE: response[0],
-                Output.MESSAGE: f"Failed to stop scan with ID '{scan_id}'",
+                Output.STATUS_CODE: 400,
+                Output.MESSAGE: f"Failed to stop scan with ID '{scan_id}'. Status of scan is {response.get('status')}",
```

Comment:
Is the status code included in the json body of the response?

Reply:
No, it never gets returned to the actions anymore

Reply:
Oh sorry 🤦 I misunderstood this.